### PR TITLE
update ucloud-uhost builder

### PR DESCRIPTION
**Pull request by shawnmssu**
IMPROVEMENTS:

- builder/ucloud-uhost: Add some access config options and run config options about `ucloud-uhost`  for improve customer experience.

### CODE REVIEW

**Comment by shawnmssu** [comment]
@SwampDragons Could you help me to resolve the conflicts. In addition, If possible, please help me review the new version docs of `ucloud-uhost` and `ucloud-import`. Thanks in advance.

File: website/pages/docs/builders/ucloud-uhost.mdx [hunk 1]
```diff
@@ -30,64 +30,27 @@ builder.
...
-- `region` - (string) This is the UCloud region. It must be provided, but it can also be sourced from the `UCLOUD_REGION` environment variables.
-
-- `availability_zone` - (string) This is the UCloud availability zone where UHost instance is located. such as: `cn-bj2-02`. You may refer to [list of availability_zone](https://docs.ucloud.cn/api/summary/regionlist)
-
-- `instance_type` - (string) The type of UHost instance. You may refer to [list of instance type](https://docs.ucloud.cn/compute/terraform/specification/instance)
-
-- `image_name` - (string) The name of the user-defined image, which contains 1-63 characters and only support Chinese, English, numbers, '-\_,.:[]'.
-
-- `source_image_id` (string) - This is the ID of base image which you want to create your customized images with.
+@include 'builder/ucloud/ImageConfig/ImageConfig-required.mdx'
```
Comment (nywilken) [inline website/pages/docs/builders/ucloud-uhost.mdx:37]:
```suggestion
@include 'builder/ucloud/common/ImageConfig-required.mdx'
```

File: website/pages/docs/builders/ucloud-uhost.mdx [hunk 2]
```diff
@@ -30,64 +30,27 @@ builder.
...
-  - `project_id` (string) - The destination project id, where copying image in.
-
-  - `region` (string) - The destination region, where copying image in.
-
-  - `name` (string) - The copied image name. If not defined, builder will use `image_name` as default name.
-
-  - `description` (number) - The copied image description.
-
-- `wait_image_ready_timeout` (int)Timeout of creating image or copying image. The default timeout is 3600 seconds if this option is not set or is set to 0.
+@include 'builder/ucloud/ImageConfig/ImageConfig-not-required.mdx'
```
Comment (nywilken) [inline website/pages/docs/builders/ucloud-uhost.mdx:45]:
```suggestion
@include 'builder/ucloud/common/ImageConfig-not-required.mdx'
```

**Comment by shawnmssu** [comment]
@nywilken Thanks for your review of this PR,  which makes me very clear on how to resolve the website failures.  Just now, I have fixed the website, but still have one single failed about `build_solaris`, if possible, please help me to resolve this problem.

File: website/pages/docs/builders/ucloud-uhost.mdx [hunk 3]
```diff
@@ -150,8 +157,59 @@ Here is a example for build UCloud Ubuntu image:
...
+build {
+  source "sources.ucloud-uhost.basic-example" {
+    image_name =  "packer-test-ubuntu{{timestamp}}"
+  }
+  provisioner "shell" {
+      execute_command = "echo '${var.password}' | sudo -S '{{.Path}}'"
+      inline          = ["sleep 30", "sudo apt update", "sudo apt install nginx -y"]
+  }
+}
+```
```
Comment (azr) [inline website/pages/docs/builders/ucloud-uhost.mdx:206]: Super nice !

File: website/pages/docs/builders/ucloud-uhost.mdx [hunk 4]
```diff
@@ -150,8 +157,59 @@ Here is a example for build UCloud Ubuntu image:
...
+  availability_zone =  "cn-bj2-02"
+  instance_type     =  "n-basic-2"
+  ssh_password      =  var.password
+  source_image_id   =  "uimage-irofn4"
+  ssh_username      =  "ubuntu"
+}
+
+build {
+  source "sources.ucloud-uhost.basic-example" {
+    image_name =  "packer-test-ubuntu{{timestamp}}"
```
Comment (azr) [inline website/pages/docs/builders/ucloud-uhost.mdx:199]: here, the hcl2 timestamp function can also be used 

```suggestion
    image_name =  "packer-test-ubuntu-${timestamp()}"
```

File: website/pages/docs/builders/ucloud-uhost.mdx [hunk 5]
```diff
@@ -107,21 +70,65 @@ Here is a basic example for build UCloud CentOS image:
...
+  region            =  "cn-bj2"
+  availability_zone =  "cn-bj2-02"
+  instance_type     =  "n-basic-2"
+  source_image_id   =  "uimage-f1chxn"
+  ssh_username      =  "root"
+}
+
+build {
+  source "sources.ucloud-uhost.basic-example" {
+    image_name =  "packer-test{{timestamp}}"
```
Comment (azr) [inline website/pages/docs/builders/ucloud-uhost.mdx:112]: Same here: 
```suggestion
    image_name =  "packer-test-${timestamp()}"
```

File: website/pages/docs/builders/ucloud-uhost.mdx (hunk 4 above)
Comment (azr) [reply website/pages/docs/builders/ucloud-uhost.mdx:199]: note that the function returns something like `2018-05-13T07:44:12Z` and I'm not sure this is allowed.

Comment (shawnmssu) [reply website/pages/docs/builders/ucloud-uhost.mdx]: @azr Just now,  I have used your suggestion about `hcl2 timestamp` to play again, that looks great! Thanks for your suggestions.

Comment (azr) [reply website/pages/docs/builders/ucloud-uhost.mdx]: super nice !

**Comment by azr** [comment]
Thanks !

**Comment by nywilken** [comment]
Just circling back to this PR. @azr took care of the merge, awesome! 
Thanks for making the update @shawnmssu.

**Pull request merged**